lib/saleorApp.ts

```diff
@@ -0,0 +1,51 @@
+import { createApp } from "../sdk/config";
```

Comment:
Here's the controversial change - instead of relying on convention (webhook query in a particular directory, with the particular name, etc.), I'm introducing conf object.

- an easy swap of the APL
- validation of the APIs/webhooks use defined APL
- manifest is now typed (but not perfect - should use enums instead of the strings)
- less magic with defining the webhooks/extensions

Open to the suggestions :)